src/python/requirements.txt

```diff
@@ -0,0 +1 @@
+tensorflow-datasets
```

Comment:
Why is this change required?

Reply:
Removing it, Good catch that got picked up accidentally.